.circleci/config.yml

```diff
@@ -39,4 +39,4 @@ workflows:
           matrix:
             parameters:
               # https://github.com/CircleCI-Public/cimg-ruby
-              ruby-version: ["2.6", "2.7", "3.0", "3.1"]
```

Comment:
Since Ruby 2.6 has been EOL for about a year now, it made sense to me to remove CI support for it.